### PR TITLE
Remove add and delete methods

### LINKED ISSUE #119: Take out add and remove from DisplayableList classes
No longer allow changing of database

### PR DESCRIPTION
**Pull request by joelngyx**
Removed add() and delete() in all classes that implement the DisplayableList interface.

Fixes #119

### CODE REVIEW

**Comment by kstonekuan** [comment]
Saw that removing these methods disrupts some Junit tests, you can either construct using an arraylist for those tests or load from the test json. If this takes too long work on the ugdg first and we can get this up a bit later!

File: src/main/java/seedu/eduke8/option/OptionList.java [hunk 1]
```diff
@@ -13,10 +13,6 @@
     private static final Logger LOGGER = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
     private static final String NO_RIGHT_ANSWER_ERROR = "Error with question: No right answer specified";
 
-    public OptionList() {
-        options = new ArrayList<>();
-    }
-
```
Comment (anqi20) [inline src/main/java/seedu/eduke8/option/OptionList.java:19]: Nice catch on removing the empty constructor too!

**Review by anqi20 — APPROVED** [review]
Everything looks awesome!

**Comment by anqi20** [comment]
Fixes #119

**Pull request merged**